src/store/reducers/cluster/cluster.ts

```diff
@@ -147,7 +147,7 @@ export function useClusterBaseInfo() {
     // Name: ydb_vla_dev02
     // Title: YDB DEV VLA02
     const clusterName = name ?? clusterNameFromQuery ?? undefined;
-    const clusterTitle = title ?? clusterName;
+    const clusterTitle = title || clusterName;
```

Comment:
Cluster `title` may be an empty string.

`??` makes `clusterTitle` an empty string but `clusterName` should be used for correct display